app-foundation/3-networks-extension/envs/production/boa_fw_rules.tf

```diff
@@ -0,0 +1,113 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  boa_gke_cluster1_master_cidr = "100.64.206.0/28"
+  boa_gke_cluster2_master_cidr = "100.65.198.0/28"
+  boa_gke_mci_master_cidr      = "100.64.198.0/28"
+  ingress_rules = {
+    "fw-${local.environment_code}-shared-base-allow-asm-healthcheck-autosidecar" = {
+      source_ranges  = [local.boa_gke_cluster1_master_cidr, local.boa_gke_cluster2_master_cidr]
+      target_tags    = ["boa-gke1-cluster", "boa-gke2-cluster"]
+      allow_protocol = "tcp"
+      allow_ports    = ["443", "10250", "15017"]
+    },
+    "fw-${local.environment_code}-shared-base-allow-8443-opa" = {
+      source_ranges  = [local.boa_gke_cluster1_master_cidr, local.boa_gke_cluster2_master_cidr]
+      target_tags    = ["boa-gke1-cluster", "boa-gke2-cluster"]
+      allow_protocol = "tcp"
```

Comment:
Since all of these FW rules are same across all env, can we put them into a module and reference them? This will also reduce maintenance burden